### DISCUSSION TITLE
auth as admin does not work correctly

### DISCUSSION THREAD

**Discussion by serkan-bayram** [pocketbase/pocketbase#4196] [Q&A]
I know I said that before "it's working now" (https://github.com/pocketbase/pocketbase/discussions/4135) but it turns out I used a hacky method to solve the issue and it still persists, now I produced a minimal example to let you try and it would be awesome **if you can take a look and feedback me as if it is a pocketbase issue or is it about nextjs.**

The problem: Usually the first function that hits authWithPassword function does not work.

![image](https://github.com/pocketbase/pocketbase/assets/68502470/ab68601b-616a-4059-866f-fe166b4b81be)

Repo: https://github.com/serkan-bayram/pocketbase-nextjs-demo

**Comment by ganigeorgiev** [comment]
The easiest way to rule out PocketBase issue is to send direct request with curl, Postman, Insomnia ,etc. Or even add a static html in `pb_public/index.html`

```html
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Example</title>
</head>
<body>
    <script type="module">
        import PocketBase from "https://cdn.jsdelivr.net/gh/pocketbase/js-sdk@0.20.3/dist/pocketbase.es.mjs"
        
        const pb = new PocketBase("http://127.0.0.1:8090");

        await pb.admins.authWithPassword("test@example.com", "1234567890")

        const records = await pb.collection("example").getFullList()
        console.log(records);
    </script>
</body>
</html>
```

In any case, please stop opening multiple issues and discussions for the same thing. I deleted your older issue for this exact reason. I posted this multiple times, double check that your problem is not caused by the Next.js cache (for how to opt out please follow the link in https://github.com/pocketbase/pocketbase/discussions/4135#discussioncomment-8113314).
